Makefile

```diff
@@ -42,6 +42,7 @@ dataset-doctest%:
 	  --ignore kedro_datasets/snowflake/snowpark_dataset.py \
 	  --ignore kedro_datasets/spark/spark_hive_dataset.py \
 	  --ignore kedro_datasets/spark/spark_jdbc_dataset.py \
+	  --ignore kedro_datasets/netcdf/netcdf_dataset.py \
```

Comment:
Why has this been ignored? Ignoring should be the exception not the rule. We've put a lot of effort trying to shorten the ignored list, so if at all possible I'd prefer if we just fix the doc string test for this new dataset.

Reply:
I'm not sure. It looks like @astrojuanlu added this while merging in main.

Reply:
I confirm locally that this was not needed 👍🏽 removing